modules/videoio/include/opencv2/videoio/videoio_c.h

```diff
@@ -560,6 +560,14 @@ CVAPI(CvVideoWriter*) cvCreateVideoWriter( const char* filename, int fourcc,
                                            double fps, CvSize frame_size,
                                            int is_color CV_DEFAULT(1));
 
+/**
+
+*/
+CVAPI(CvVideoWriter*) cvCreateVideoWriterWithPreference(int api, const char* filename, int fourcc,
+                                           double fps, CvSize frame_size,
+                                           int is_color CV_DEFAULT(1));
```

Comment:
Probably we should not add new exported `cv*` functions. Please make it local "static".